src/plugins/intel_cpu/tests/functional/shared_tests_instances/snippets/mha.cpp

```diff
@@ -68,6 +68,12 @@ std::vector<std::vector<InputShape>> transposedShape_3D(bool with_dynamic = true
     return shapes;
 }
 
+std::vector<std::vector<InputShape>> transposedShape_2D(bool with_dynamic = true) {
+    auto shapes = SNIPPETS_TESTS_STATIC_SHAPES(
```

Comment:
Please add dynamic shapes. Or we can add only dynamic instance and remove `with_dynamic` param

Reply:
Done